src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -78,6 +85,39 @@ public Response createDaaForDac(
     }
   }
 
+  @POST
```

Comment:
The more I think about this, the more I think it should be a `PUT`, not a `POST`. We're not creating any new objects here, just modifying a relationship between two existing ones.